src/commons/listener.rs

```diff
@@ -31,6 +31,8 @@ pub enum ServiceType {
     NodePort,
     /// Provision a dedicated load balancer.
     LoadBalancer,
+    /// Generate a cluster IP.
```

Comment:
Copied from k8s docs ^^
```suggestion
    /// Assigns an IP address from a pool of IP addresses that your cluster has reserved for that purpose.
```